### PR TITLE
Version 13.0.0 → 14.0.0

### PR DESCRIPTION
**Pull request by Gabriel439**

### CODE REVIEW

**Comment by philandstuff** [comment]
Technically, #898 is a breaking change, even if the likelihood of breakage is small...

**Comment by sjakobi** [comment]
Should we use the changelog to announce the deprecation of `Optional/{build,fold}` (https://github.com/dhall-lang/dhall-lang/pull/880)? Not sure how this was handled with previous deprecations…

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,132 @@ file.
 
 For more info about our versioning policy, see [versioning.md](standard/versioning.md).
 
+## `v14.0.0`
+
+Deprecation notice:
+
+* [The `Optional/fold` and` `Optional/build` built-ins are deprecated](https://docs.dhall-lang.org/howtos/migrations/Deprecation-of-Optional-fold-and-Optional-build.html)
+
+  These built-ins will be removed in the following release in an effort to
```
Comment (sjakobi) [inline CHANGELOG.md]: Actually the built-ins are scheduled for removal in v17.

File: CHANGELOG.md [hunk 2]
```diff
@@ -5,6 +5,132 @@ file.
...
+  { a = { b = { c = 1, d = True } } }
+  ```
+
+* [Fix typing rule for merging `Optional`s](https://github.com/dhall-lang/dhall-lang/pull/899)
+
+  The previous released introduced `merge` support for `Optional` values, such
+  as:
+
+  ```dhall
+  merge { None = 0, Some = λ(n : Natural) → n } (Some 4)
```
Comment (TristanCacqueray) [inline CHANGELOG.md]: I find this much easier to use than `Optional/fold`, thanks!

**Comment by philandstuff** [comment]
Could we also announce deprecation of urls with quoted paths? That’s the next step for #581.

**Comment by Gabriel439** [comment]
@philandstuff: I think we need a Migration page first before we can announce deprecation, mainly so that the announcement has something to link to

**Comment by philandstuff** [comment]
Ah ok. I can write one, and I might also add this process to versioning.md because that’s where I looked to try to find a depreciation process before. (Don’t hold the release up for it though, obviously)

**Pull request merged**